README.md

```diff
@@ -137,8 +137,8 @@ anywhere.
 In other words, kube-state-metrics itself is designed to be another source for
 Heapster (although this is not currently the case).
 
-Additionally, some monitoring systems such as Prometheus do not use Heapster
-for metric collection at all and instead implement their own. Having
+Additionally, some monitoring systems such as Prometheus can use both Heapster
+and kube-state-metrics as scrape target for metric collection. Having
```

Comment:
As far as I know heapster (as of today) does not expose a metrics endpoint for Prometheus to scrape so this is also not 100% correct.

What @fabxc was talking about on the mailing list was, that there are reasons to run both Prometheus and Heapster in your cluster. Heapster is required for autoscaling with the horizontal pod autoscaler. Heapster mainly collects lower level container metrics. Prometheus is for all kinds of metrics and it also has alerting features.

So you want to actually run both in your cluster today:

* Heapster for metric collection and autoscaling
* Prometheus for metric collection, alerting and dashboarding

Eventually as the metrics API becomes available in Kubernetes, we will be able to get rid of Heapster and replace it with a Prometheus based proxy that will be drop-in.

/cc @fabxc